### PR TITLE
[TASK] Show error on wrong regular key (DC-107)

### PR DESCRIPTION
**Pull request by MatthewPhinney**

### CODE REVIEW

File: src/js/tabs/coldwallet.js [hunk 1]
```diff
@@ -71,10 +71,10 @@ ColdWalletTab.prototype.angular = function (module) {
       return txnMessage;
     }*/
 
-    var address = $routeParams.address;
+    var address = $scope.address = $routeParams.address;
```
Comment (mrajvanshy) [inline src/js/tabs/coldwallet.js]: This is already fixed on the release branch.

Comment (MatthewPhinney) [inline src/js/tabs/coldwallet.js]: I'll update the PR.

**Comment by MatthewPhinney** [comment]
@mrajvanshy I merged release back to develop and updated the PR. I think it should be OK now. Nice catch!

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**